File: mentora-web/src/app/services/responses/course-slide.response.ts
export interface CourseSlideResponse {
  id: string;
  courseId: string;
  courseName: string;
  slideTypeId: string;
  slideTypeName: string;
  title: string;
  content: string;
  ordering: number;
  active: boolean;
  createdAt: string;
  updatedAt: string;
}

File: server/src/Mentora.Application/DTOs/CourseSlideResponse.cs
namespace Mentora.Application.DTOs;

public record CourseSlideResponse
{
    public Guid Id { get; set; }
    public Guid CourseId { get; set; }
    public string CourseName { get; set; } = string.Empty;
    public Guid SlideTypeId { get; set; }
    public string SlideTypeName { get; set; } = string.Empty;
    public string Title { get; set; } = string.Empty;
    public string Content { get; set; } = string.Empty;
    public int Ordering { get; set; }
    public bool Active { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
}
